classes/task/get_meeting_recordings.php

```diff
@@ -78,44 +78,76 @@ public function execute() {
             'video' => get_string('recordingtypevideo', 'mod_zoom'),
         ];
 
-        $zoommeetings = zoom_get_all_meeting_records();
-        foreach ($zoommeetings as $zoom) {
+        $localmeetings = zoom_get_all_meeting_records();
+
+        $now = time();
+        $from = gmdate('Y-m-d', strtotime('-1 day', $now));
+        $to = gmdate('Y-m-d', $now);
```

Comment:
Include recordings that completed today.
```suggestion
        $to = gmdate('Y-m-d', strtotime('+1 day', $now));
```